src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/domain/event/HmppsPeriodLengthMessage.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.domain.event
+
+data class HmppsPeriodLengthMessage(
```

Comment:
was wondering whether we need all this meta data.. or would periodLengthId and source suffice?
I think it would be useful to have all the data for diagnostics though.